src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -169,7 +170,7 @@ def _get_revert_count(self, schema: str | None = None, table: str | None = None)
         return migration_list
 
     def is_upgraded(self, schema: str, table: str) -> bool:
-        result = self._backend.fetch(f"SHOW TBLPROPERTIES `{schema}`.`{table}`")
+        result = self._backend.fetch(f"SHOW TBLPROPERTIES {escape_sql_identifier(schema+'.'+table)}")
```

Comment:
You need to escape two identifiers

Reply:
`escape_sql_identifier` automatically handles identifiers with multiple parts